packages/upstream/version.txt

```diff
@@ -0,0 +1,3 @@
+# Specify a version of protovalidate with a leading "v", for example v0.13.1.
+# Alternatively, specify a branch name on the main protovalidate repository.
+v0.13.3
```

Comment:
The new single source of truth for the upstream version. We can split this up into separate refs for protovalidate and protovalidate-testing, but I don't see the necessity right now.